eng/versioning/version_client.txt

```diff
@@ -112,6 +112,7 @@ com.azure.spring:azure-spring-cloud-appconfiguration-config-web;1.2.8-beta.1;1.2
 com.azure.spring:azure-spring-cloud-appconfiguration-config;1.2.8-beta.1;1.2.8-beta.1
 com.azure.spring:azure-spring-cloud-feature-management-web;1.2.8-beta.1;1.2.8-beta.1
 com.azure.spring:azure-spring-cloud-feature-management;1.2.8-beta.1;1.2.8-beta.1
+com.azure.spring:azure-spring-cloud-starter-appconfiguration-config;1.2.8-beta.1;1.2.8-beta.1
```

Comment:
Please restore the contents deleted in this PR: https://github.com/Azure/azure-sdk-for-java/pull/19033/files

Reply:
And this one: https://github.com/Azure/azure-sdk-for-java/pull/18945/files